pylinalg/func/vector.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+
+def vector_add_vector(v1, v2, out=None):
+    if out is None:
+        out = np.empty_like(v1 if v1.size > v2.size else v2)
+    out[:] = v1 + v2
+    return out
+
+
+def vector_sub_vector(v1, v2, out=None):
+    if out is None:
+        out = np.empty_like(v1 if v1.size > v2.size else v2)
+    out[:] = v1 - v2
```

Comment:
can I call np.sub(v1, v2, out=out) ?

Reply:
resolved this by just making vector_sub_vector an alias for np.subtract since it has exactly the same signature